avaframe/runCom1DFA.py

```diff
@@ -20,7 +20,7 @@
 avalancheDir = cfgMain['MAIN']['avalancheDir']
 
 # Clean input directory of old work and output files from module
-initProj.cleanModuleFiles(avalancheDir, com1DFA, deleteOutput=True)
+initProj.cleanModuleFiles(avalancheDir, com1DFA, deleteOutput=False)
```

Comment:
Mh, I think the default here should be to delete the output. Did you need this for development or is there a reason to switch this generally?